Sources/Validation/Validators/RangeValidator.swift

```diff
@@ -70,13 +70,13 @@ fileprivate struct RangeValidator<T>: ValidatorType where T: Comparable {
     func validate(_ data: T) throws {
         if let min = self.min {
             guard data >= min else {
-                throw BasicValidationError("is not larger than \(min)")
+                throw BasicValidationError("is less than \(min)")
```

Comment:
Same issue here I believe
```suggestion
                throw BasicValidationError("is not greater than \(min)")
```

Reply:
Actually, I believe the current message `"is not larger"` is incorrect as `min` is a valid value.

Reply:
Again as above, I'm a fan of using "less than" and "greater than" for numeric values... and in this `RangeValidator` case I believe @baarde is correct.

(Again too, for extra error clarity you might also consider more verbose forms along the lines of `is less than required minimum of \(min)` and `is greater than required maximum of \(max)`.)